tmpfiles.d/legacy.conf.in

```diff
@@ -12,7 +12,7 @@
 
 d /run/lock 0755 root root -
 L /var/lock - - - - ../run/lock
-{% if CREATE_LOG_DIRS %}
+{% if HAVE_SYSV_COMPAT and CREATE_LOG_DIRS %}
```

Comment:
Hmm, please rebase on main, and wrap this in the HAVE_SYSV_COMPAT block directly.

Reply:
Ah, had been wondering why this did not look exactly as expected. 🤪

Rebased, updated and pushed.

(Time for the next coffee ☕️ I guess.)